airflow/config_templates/config.yml

```
@@ -1175,6 +1175,19 @@
       type: string
       example: ~
       default: ""
+- name: celery_kubernetes_executor
+  description: |
+    This section only applies if you are using the CeleryKubernetesExecutor in
+    ``[core]`` section above
+  options:
+    - name: kubernetes_queue
+      description: |
+        Define when to use KubernetesExecutor for a task. When the queue of a task is kubernetes_queue,
```

```suggestion
        Define when to send a task to KubernetesExecutor when using CeleryKuebernetesExecutor. When the queue of a task is kubernetes_queue,
```
But no strong opinion